.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,44 +0,0 @@
-<component name="InspectionProjectProfileManager">
```

Comment:
🤔 I have some doubt about removing this file based on [this](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems)

Reply:
I know that I didn't make any changes to our project config that should be shared, yet those files were modified. If this is some automated stuff by AS, I see no reason to share it. I haven't dug deep into it though...